### ISSUE TITLE
Babel support?

### ISSUE THREAD

**Issue by corysimmons** [roadhump/SublimeLinter-eslint#123]
Errors on any es6. Here's "illegal import declaration".

![image](https://cloud.githubusercontent.com/assets/224636/13099618/17dc654c-d502-11e5-843e-4ee5bdbafb49.png)

I don't have an .eslintrc file. I prefer not to pollute my projects with 50 config files, instead I put that type of config in package.json

![image](https://cloud.githubusercontent.com/assets/224636/13099626/222ed66a-d502-11e5-9660-a79e966dcaa3.png)

I haven't tweaked anything in SublimeLinter settings for eslint or standard.

![image](https://cloud.githubusercontent.com/assets/224636/13099647/594f0638-d502-11e5-95c3-40bd807e154e.png)

![image](https://cloud.githubusercontent.com/assets/224636/13099655/77bd6632-d502-11e5-9aff-97c44a5d8f5d.png)

**Comment by roadhump** [comment]
@corysimmons Does `eslint` from command line work?

**Comment by jcolemorrison** [comment]
Same here, it doesn't recognize import for es6.  And `eslint` is working on the command line for me.

**Comment by jcolemorrison** [comment]
Actually, adding a bunch of rules fixed it as outlined in this issue:

https://github.com/roadhump/SublimeLinter-eslint/issues/91

it was the `"no-reserved-keys"` property, which I suppose by default is turned to where es6 keywords aren't allowed.

**Comment by corysimmons** [comment]
@roadhump You're right. I'm an idiot. Sorry to have bothered you.

For others trying to use package.json config with SublimeLinter-contrib-eslint:

```
{
  "eslintConfig": {
    "parser": "babel-eslint"
  },
  "devDependencies": {
    "babel-eslint": "^5.0.0",
    "eslint": "^2.2.0"
  }
}
```

**Comment by a-x-** [comment]
I have a similar issue.

```js
class Foo {
  bar = e => e.baz()
}
```

`SublimeLinter: eslint: Error - Parsing error: Unexpected token =`
